orca-core/src/main/groovy/com/netflix/spinnaker/orca/ExecutionStatus.java

```diff
@@ -79,31 +80,40 @@
   /**
    * The task was skipped and the pipeline will proceed to the next task.
    */
-    SKIPPED(true, false)
+    SKIPPED(true, false);
 
   /**
    * Indicates that the task/stage/pipeline has finished its work (successfully or not).
    */
-  final boolean complete
+  public final boolean isComplete() {
+    return complete;
+  }
 
   /**
    * Indicates an abnormal completion so nothing downstream should run afterward.
    */
-  final boolean halt
+  public final boolean isHalt() {
+    return halt;
+  }
+
+  private static final Collection<ExecutionStatus> SUCCESSFUL = Collections.unmodifiableList(Arrays.asList(SUCCEEDED, STOPPED, SKIPPED));
```

Comment:
possibly, probably?

I just groovy->java'd this so I wasn't really looking at the semantics of this class

I don't really want to add that change into this PR since it feels unrelated